lib/web/apiserver_test.go

```diff
@@ -4700,14 +4707,20 @@ func TestGetWebConfig_WithEntitlements(t *testing.T) {
 	expectedCfg.Entitlements[string(entitlements.JoinActiveSessions)] = webclient.EntitlementInfo{Enabled: false}
 	expectedCfg.Entitlements[string(entitlements.K8s)] = webclient.EntitlementInfo{Enabled: false}
 
-	// request and verify enabled features are enabled.
-	re, err = clt.Get(ctx, endpoint, nil)
-	require.NoError(t, err)
-	require.True(t, strings.HasPrefix(string(re.Bytes()), "var GRV_CONFIG"))
-	str = strings.ReplaceAll(string(re.Bytes()), "var GRV_CONFIG = ", "")
-	err = json.Unmarshal([]byte(str[:len(str)-1]), &cfg)
-	require.NoError(t, err)
-	require.Equal(t, expectedCfg, cfg)
+	// request and verify enabled features are eventually enabled.
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		re, err = clt.Get(ctx, endpoint, nil)
+		require.NoError(t, err)
+		require.True(t, strings.HasPrefix(string(re.Bytes()), "var GRV_CONFIG"))
+		str = strings.ReplaceAll(string(re.Bytes()), "var GRV_CONFIG = ", "")
+		err = json.Unmarshal([]byte(str[:len(str)-1]), &cfg)
+		require.NoError(t, err)
+		diff := cmp.Diff(expectedCfg, cfg)
+		if assert.Empty(c, diff) {
+			t.Logf("Feature diff (-want +got):\n%s", diff)
+		}
+
+	}, time.Second, time.Millisecond*50)
```

Comment:
I'd bump this up to 3 or 5 seconds, 1 second tends to be flaky on our slow GHA runners.

Reply:
I've bumped it to 5 seconds, thanks.